frontend/src/components/CategoryBar.css

```diff
@@ -41,13 +46,18 @@
   color: white;
 }
 
-/* Desktop styles */
-@media (min-width: 768px) {
+@media (max-width: 768px) {
   .category-bar {
-    flex-direction: column;
-    top: 0;
-    left: 0;
-    padding: 3em;
-    gap: 9rem;
+    display: flex;
```

Comment:
Don't Repeat Yourself : what was said by default does'nt have to be said again in a media query